### DISCUSSION TITLE
Gaggia Baby Twin

### DISCUSSION THREAD

**Discussion by lee-vee-o** [Zer0-bit/gaggiuino#486] [Q&A] [answered]
This is an amazing project! I’d love to use it for my espresso machine, the Gaggia Baby Twin, whose PCB doesn’t work any more. However, the configuration is different than the Gaggia Classic, as it has a rapid steamer and the steam release is controlled through an extra PCB and a dedicated steam valve. 
What would it take to adapt the project to this model too? 
I’d be happy to help, though I reckon I have limited knowledge to do it myself.

**Comment by banoz** [comment, marked as answer]
PCB v2 is the solution. It has everything you need to control such machine. Join the discord and create your install-help thread to get the support.

**Reply by lee-vee-o** [reply]
That’s great news, thanks @banoz! I just joined the discord, I’ll create the install-help thread as you suggest.
